### PR TITLE
[1.59.0] add docs for 'get config'

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

**Comment by laverya** [comment]
docs for https://github.com/replicatedhq/kots/pull/2368

File: content/kots-cli/get/config.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+kubectl kots get config [flags]
+```
+
+- _Provide `[flags]` according to the table below_
+
+| Flag              | Type   | Description                                                         |
+| :---------------- | ------ | ------------------------------------------------------------------- |
+| `--appslug`       | string | app slug to retrieve config for _(required)_                        |
+| `--sequence`      | int    | app sequence to retrieve config for _(required)_                    |
+| `-h, --help`      |        | help for get apps                                                   |
```
Comment (samantha-replicated) [inline content/kots-cli/get/config.md:22]:
```suggestion
| `-h, --help`      |        | The help for the `get apps` command                                                  |
```

File: content/kots-cli/get/config.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+| `-n, --namespace` | string | the namespace where the admin console is running _(required)_       |
```
Comment (samantha-replicated) [inline content/kots-cli/get/config.md:23]:
```suggestion
| `-n, --namespace` | string | The namespace where the admin console is running _(required)_       |
```

**Review by samantha-replicated — CHANGES_REQUESTED** [review]
Hey! I have a few minor edits. Feel free to commit my suggestions if they are accurate from a technical standpoint. Thanks!

File: content/kots-cli/get/config.md (hunk 1 above)
Comment (laverya) [reply content/kots-cli/get/config.md:22]: not touching this, as it's shared with a bunch of other commands
fixing them all at once might be a good idea though

File: content/kots-cli/get/config.md (hunk 2 above)
Comment (laverya) [reply content/kots-cli/get/config.md:23]: same here - `--namespace` is a pretty common flag

**Pull request merged**